pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_delete-modules.adoc

```diff
@@ -0,0 +1,18 @@
+[id="delete-modules_{context}"]
+
+= Deleteing a module in {ProductShortName}
+
+Deleting a module in {ProductShortName}.
+
+.Prerequisites
+
+//* You are assigned a publisher or admin role in {ProductShortName}.
+* You are logged in.
+
+.Procedure
+
+. Navigate to the link:{LinkToSearchPage}[Search page].
+. Select the check box of the module you want to delete.
+. Click *Delete*.
+. In the confirmation screen, click *OK* to confirm.
+. The module is deleted, and is no longer available in the *Modules* tab.
```

Comment:
As this is not a step, I suggest not having it as a list item. Perhaps it could be aligned with the last step as 

. In the confirmation screen, click *OK* to confirm.
+
The module is deleted and is no longer available in the *Modules* tab.

Or,

as a standalone statement 

. In the confirmation screen, click *OK* to confirm.

The module is deleted and is no longer available in the *Modules* tab.

The latter structure would serve as the result for the module - I prefer this!